coursemapper-kg/recommendation/app/services/course_materials/recommendation/youtube_service.py

```
@@ -42,17 +43,65 @@
+        raise Exception("All API keys have exceeded their quota.")
+
 
     def get_videos(self, concepts, top_n=15):
         logger.info("Get Videos")
-        logger.info(concepts)
+        # logger.info(concepts)
         video_data = []
         duration_list = []
         view_list = []
         description_list = []
         retry_count = 3
```

Ruff (0.12.2)

Local variable `retry_count` is assigned to but never used

Remove assignment to unused variable `retry_count`

(F841)